lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.html

```diff
@@ -4,8 +4,14 @@
             class="adf-option-chips"
             *ngFor="let option of selectedOptions"
             (removed)="remove(option)">
-            <span>{{option}}</span>
-            <button matChipRemove class="adf-option-chips-delete-button" [attr.aria-label]="('SEARCH.FILTER.BUTTONS.REMOVE' | translate) + ' ' + option">
+            <span [matTooltip]="'SEARCH.RESULTS.WILL_CONTAIN' | translate:{searchTerm: option.fullPath}"
```

Comment:
I think it should take `fullPath` and if it's not present it should display `value`